### PR TITLE
feat(console): page error state

### PR DESCRIPTION
**Pull request by xiaoyijun**
## Summary

Add page error state to details page.

## Testing

image

image

image

### CODE REVIEW

File: packages/phrases/src/locales/de/translation/admin-console/errors.ts [hunk 1]
```diff
@@ -16,6 +16,7 @@ const errors = {
   password_pattern_error: 'Das Passwort muss aus mindestens 6 Zeichen lang sein',
   insecure_contexts: 'Unsichere Kontexte (nicht-HTTPS) werden nicht unterstützt.',
   unexpected_error: 'Ein unerwarteter Fehler ist aufgetreten',
+  not_found: '404 not found',
```
Comment (charIeszhao) [inline packages/phrases/src/locales/de/translation/admin-console/errors.ts:19]: nit: Add `// UNTRANSLATED`

File: packages/console/src/components/RequestDataError/index.tsx [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+  const errorMessage = error.body?.message ?? error.message;
+  const isNotFoundError = error.status === 404;
+
+  return (
+    <Card className={classNames(styles.error, className)}>
+      {theme === AppearanceMode.LightMode ? (
+        <RequestErrorImage className={styles.image} />
+      ) : (
+        <RequestErrorDarkImage className={styles.image} />
+      )}
```
Comment (charIeszhao) [inline packages/console/src/components/RequestDataError/index.tsx:32]: You can select the component first and then render it, in which way you can avoid defining the `className` twice. E.g.:
```ts
const Image = theme === AppearanceMode.LightMode ? RequestErrorImage : RequestErrorDarkImage;
...
<Image className={styles.image} />
```

File: packages/phrases/src/locales/de/translation/admin-console/errors.ts (hunk 1 above)
Comment (xiaoyijun) [reply packages/phrases/src/locales/de/translation/admin-console/errors.ts]: Updated.

File: packages/console/src/components/RequestDataError/index.tsx (hunk 2 above)
Comment (xiaoyijun) [reply packages/console/src/components/RequestDataError/index.tsx]: Updated.

**Review by charIeszhao — APPROVED** [review]

**Pull request merged**